src/illuminatio/cleaner.py

```diff
@@ -16,59 +22,86 @@ def __init__(self, core_api, apps_api, rbac_api, logger=None):
         self.rbac_api = rbac_api
 
     def clean_up_pods_in_namespaces(self, namespaces, cleanup_policy):
-        return self.delete_res_using(namespaces, cleanup_policy, self.core_api.delete_collection_namespaced_pod, "pod")
+        """
+        Deletes all pods in a list of namespaces
+        """
+        return self.delete_resource_with_cleanup_policy(namespaces, cleanup_policy,
+                                                        self.core_api.delete_collection_namespaced_pod, "pod")
 
     def clean_up_services_in_namespaces(self, namespaces, cleanup_policy):
-        def delete_collection_namespaced_serivce(namespace, label_selector=None):
+        """
```

Comment:
Why is the service clean up rather complex compared to the other clean up methods ?

Reply:
Unfortunately `delete_namespaced_service` does not provide the convenience to use a label selector. `delete_collection_namespaced_service` covers this issue as a wrapper method

Reply:
I added a comment

Reply:
Thanks